<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeSelectClass</class>
 <widget class="QWidget" name="ModeSelectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color: rgb(255, 255, 255);	
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 116, 174);
border-radius: 15px;   </string>
    </property>
    <property name="text">
     <string>场景选择</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>700</y>
      <width>220</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>971</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_dubug_mode">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/mode_select/调试.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/mode_select/选中.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 14pt &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>调试</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_work_mode">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/mode_select/作业.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/mode_select/选中.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 14pt &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>作业</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_service_mode">
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/mode_select/远程.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/mode_select/选中.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 14pt &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>远程</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
